src/consensus/aft/raft.h

```diff
@@ -2800,17 +2800,6 @@ namespace aft
     {
       std::lock_guard<std::mutex> guard(state->lock);
 
-      // Ignore if we don't recognise the node.
```

Comment:
Could we add a comment here, stating that it's OK to accept votes from unknown nodes?